lib/l10n/intl_en.arb

```diff
@@ -3,6 +3,13 @@
   "appInfoTechnicalDescription": "Application fully written in Flutter",
   "appInfoFeatureDescription": "The latest news from the world",
   "emailed": "E-mailed",
+  "location": "Location",
+  "yourLocation": "Your location",
+  "latitude": "Latitude",
+  "longitude": "Longitude",
+  "permissionDeniedForeverErrorMessage": "Location service is not available. Please go to the settings to grant the permission.",
+  "permissionDeniedErrorMessage": "Permission denied. Click the button to retry action.",
```

Comment:
```suggestion
  "locationPersmissionDeniedForeverErrorMessage": "Location service is not available. Please go to the settings to grant the permission.",
  "permissionDeniedErrorMessage": "Permission denied. Click the button to retry action.",
```
ten drugi jest generyczny

Reply:
mógłbyś precyzyjniej? :P

Reply:
`permissionDeniedForeverErrorMessage` - ta nazwa nie mówi nic, że to dotyczy permissionu lokalizacji, a string tylko tego permissionu będzie dotyczyć